lib/ExpensiMark.js

```diff
@@ -370,7 +405,9 @@ export default class ExpensiMark {
             }
             generatedMarkdown = generatedMarkdown.replace(rule.regex, rule.replacement);
         });
-        return Str.htmlDecode(Str.stripHTML(generatedMarkdown));
+        generatedMarkdown = generatedMarkdown
+            .replace(/<div.*?>|<\/div>|<comment.*?>|\n<\/comment>|<\/comment>|<h1>|<\/h1>|<h2>|<\/h2>|<h3>|<\/h3>|<h4>|<\/h4>|<h5>|<\/h5>|<h6>|<\/h6>/gm, '[block]');
```

Comment:
> this is the final code do you think we should push it or keep the old one. ?

Actually I meant that we just store indices for this. Instead of replacing it with '[block]'
But it may be more trouble than it's worth. So let's do nothing.

Reply:
ok,
we not using the index here because we will split the HTML tags and the index will change with every string, we split it. 
also when we add a new line the index will change. 

thanks.